frontend/src/components/layout/Layout.tsx

```diff
@@ -10,8 +10,8 @@ export const Layout = ({
   username,
 }: {
   children: ReactElement;
-  authenticated?: boolean | undefined;
   username?: string;
+  authenticated?: boolean;
```

Comment:
This is a sonarqube fix?
Couldn't we tell `authenticated` by the presence of a username?

Reply:
This isn't a new line, not sure why the whole line is showing as new. The fix was for a redundant | undefined since the authenticated is already set to optional with the "?" mark. 

![Screenshot 2024-01-31 at 11 33 09 AM](https://github.com/bcgov/talent-cloud/assets/78890675/73577351-efea-4efc-b7e6-fe79437e263d)